pkgs/development/python-modules/qemu/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchurl
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "qemu";
+  qemuVersion = "7.0.0";
```

Comment:
Looks like that if we don't initiate from `qemu` then it requires manual intervention to keep it in sync. 

Have you thought about add the building of the bindings as part of the `[qemu](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/virtualization/qemu/default.nix)` package?

Reply:
Hmmm good point. I've updated the package to follow the [capstone bindings](https://github.com/NixOS/nixpkgs/blob/d6240ce11a56c6e9daee6213c93cecd86ba58ea7/pkgs/development/python-modules/capstone/default.nix#L12) which references the main capstone package's `src`. The only issue is that the Python bindings for QEMU seem to have a separate version embedded in [this upstream file](https://github.com/qemu/qemu/blob/master/python/VERSION). The bindings aren't published on PyPI unfortunately.

I also added some of the optional dependencies for the bindings declared in the [upstream `setup.cfg`](https://github.com/qemu/qemu/blob/master/python/setup.cfg#L34).